packages/orm/src/drivers/buildRawQuery.ts

```diff
@@ -0,0 +1,161 @@
+import { Knex } from "knex";
+import { opToFn } from "../EntityGraphQLFilter";
+import { isDefined } from "../EntityManager";
+import { ColumnCondition, ParsedExpressionFilter, ParsedFindQuery, ParsedTable, RawCondition } from "../QueryParser";
+import { kq, kqDot } from "../keywords";
+import { assertNever, fail } from "../utils";
+import QueryBuilder = Knex.QueryBuilder;
+
+/**
+ * Transforms `ParsedFindQuery` into a raw SQL string.
+ *
+ * In theory this should be implemented within each Driver, but the logic will be largely
+ * the same for different dbs.
+ */
+export function buildRawQuery(
+  parsed: ParsedFindQuery,
+  settings: { limit?: number; offset?: number },
+): { sql: string; bindings: readonly any[] } {
```

Comment:
This is a copy/paste of `buildKnexQuery`, with the knex calls rewritten to string building.